tests/snippets/stdlib_math.py

```diff
@@ -240,6 +240,15 @@ def assertAllNotClose(examples, *args, **kwargs):
 assert math.factorial(20) == 2432902008176640000
 assert_raises(ValueError, lambda: math.factorial(-1))
 
+assert math.nextafter(4503599627370496.0, -INF) == 4503599627370495.5
```

Comment:
if this is 3.9 feature, CPython in CI will not run this test. Adding version check will work

Reply:
Oh, thanks, this is 3.9 feature, I will add it